sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchAsyncClient.java

```diff
@@ -187,12 +187,12 @@ public SearchBatchAsyncClient getSearchIndexDocumentBatchingAsyncClient() {
      * @param batchSize The number of documents in a batch that will trigger it to be indexed. If automatic batch
      * sending is disabled this value is ignored. If {@code batchSize} is null a default value of 1000 is used.
      * @param indexingHook An implementation of {@link IndexingHook} used to handle document callback actions.
-     * @return A {@link SearchBatchAsyncClient} used to index documents for the Search index associated with
-     * this {@link SearchAsyncClient}.
+     * @return A {@link SearchBatchAsyncClient} used to index documents for the Search index associated with this {@link
+     * SearchAsyncClient}.
      * @throws IllegalArgumentException If {@code batchSize} is less than one.
      */
-    public SearchBatchAsyncClient getSearchIndexDocumentBatchingAsyncClient(Boolean autoFlush,
-        Duration flushWindow, Integer batchSize, IndexingHook indexingHook) {
+    public SearchBatchAsyncClient getSearchBatchAsyncClient(Boolean autoFlush, Duration flushWindow, Integer batchSize,
+        IndexingHook indexingHook) {
```

Comment:
Do you think we could reorder the params like 
`IndexingHook indexingHook, Boolean autoFlush, Duration flushWindow, Integer batchSize,
        Duration flushWindow, Integer batchSize) `

Reply:
>     * @throws IllegalArgumentException If {@code batchSize} is less than one.
Keeping this in mind can batchSize be `int` instead?

Reply:
`Integer` is used as there is a default value.